_examples/mold/main.go

```diff
@@ -35,7 +35,7 @@ func main() {
 	fmt.Println(myString)
 }
 
-func transformMyData(ctx context.Context, t *mold.Transformer, value reflect.Value, param string) error {
+func transformMyData(_ context.Context, _ *mold.Transformer, value reflect.Value, _ string) error {
```

Comment:
I would like to keep the unused parameter names intact for the examples to drive clarity of what they mean, but the rest of the changes look good.

Reply:
I agree. Done.